File: src/main/java/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsDto.java
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;
import uk.co.threebugs.mochiwhattotrade3.account.AccountDto;
import uk.co.threebugs.mochiwhattotrade3.setupgroups.SetupGroupsDto;

@Value
@Builder
@Jacksonized
public class AccountSetupGroupsDto {

    Integer id;
    String name;
    AccountDto account;
    SetupGroupsDto setupGroups;
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsDto.kt
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups

import uk.co.threebugs.mochiwhattotrade3.account.AccountDto
import uk.co.threebugs.mochiwhattotrade3.setupgroups.SetupGroupsDto


data class AccountSetupGroupsDto (
    var id: Int? = null,
    var name: String? = null,
    var account: AccountDto? = null,
    var setupGroups: SetupGroupsDto? = null
)

